todos/src/components/Task.js

```diff
@@ -11,9 +11,36 @@ const Task = (props) => {
   const currentData = myList.data;
   const onEdit = myList.onEdit;
 
+  const completedCardLoadingState = useSelector(
+    (state) => state.laodingReducer.completedCardLoadingState
+  );
+
+  const deleteCardLoadingState = useSelector(
+    (state) => state.laodingReducer.deleteCardLoadingState
+  );
+
+  const currentSelectedId = useSelector(
+    (state) => state.laodingReducer.currentSelectedId
+  );
+
   return (
     <div>
-      {onEdit ? (
+      {(completedCardLoadingState ||
+      deleteCardLoadingState) &&
+      currentSelectedId === props.id ? (
```

Comment:
Can we use computed property here?

Reply:
Yes Bhai, we can use computed property to make the code more concise and easier to read. Doing it right now.